README.MD

```diff
@@ -34,8 +34,17 @@ Presidio can be integrated into any data pipeline for intelligent PII scrubbing.
 - [Presidio Input and Output](docs/overview.md#input-and-output)
 - [API Spec](presidio-api/cmd/presidio-api/docs/readme.md)
 - [The Technology Stack](docs/overview.md#the-technology-stack)
+- [Architecture](docs/design.md)
+
+## Presidio API
+
+Presidio API is served using the [presidio-api](presidio-api) golang service.
```

Comment:
I think this can be simplified by removing the first two lines. Simply the presidio API is described ... 
these introduce already the architecture and networking of presidio, which is too much info at this level